lib/barcodevalidation/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module BarcodeValidation
-  VERSION = "2.2.0".freeze
+  VERSION = "2.2.0"
```

Comment:
Not familiar with the release process, but assuming I'll have to bump this to 2.3.0?